service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -862,42 +917,9 @@ private void addWsmResourceOwnerPolicy(CreateResourceRequestV2 request)
     }
   }
 
-  /**
-   * Add policies for managing private users via policy. All private resources will have reader,
-   * writer, and editor policies even if they are empty to support potential later reassignment.
-   * This method will likely expand to support policies for applications in the future.
-   */
-  private void addPrivateResourcePolicies(
-      CreateResourceRequestV2 request,
-      List<ControlledResourceIamRole> privateIamRoles,
-      String privateUser) {
-    AccessPolicyMembershipV2 readerPolicy =
-        new AccessPolicyMembershipV2().addRolesItem(ControlledResourceIamRole.READER.toSamRole());
-    AccessPolicyMembershipV2 writerPolicy =
-        new AccessPolicyMembershipV2().addRolesItem(ControlledResourceIamRole.WRITER.toSamRole());
-    AccessPolicyMembershipV2 editorPolicy =
-        new AccessPolicyMembershipV2().addRolesItem(ControlledResourceIamRole.EDITOR.toSamRole());
-
-    // Create a reader or writer role as specified by the user request, but also create empty
-    // roles in case of later re-assignment.
-    if (privateIamRoles.contains(ControlledResourceIamRole.WRITER)) {
-      writerPolicy.addMemberEmailsItem(privateUser);
-    } else if (privateIamRoles.contains(ControlledResourceIamRole.READER)) {
-      readerPolicy.addMemberEmailsItem(privateUser);
-    }
-
-    if (privateIamRoles.contains(ControlledResourceIamRole.EDITOR)) {
-      editorPolicy.addMemberEmailsItem(privateUser);
-    }
-
-    request.putPoliciesItem(ControlledResourceIamRole.READER.toSamRole(), readerPolicy);
-    request.putPoliciesItem(ControlledResourceIamRole.WRITER.toSamRole(), writerPolicy);
-    request.putPoliciesItem(ControlledResourceIamRole.EDITOR.toSamRole(), editorPolicy);
-  }
-
   /**
    * Fetch the email of a user's pet service account in a given project. This request to Sam will
-   * create the pet SA if it doesn't already exist.
+   * crea44te the pet SA if it doesn't already exist.
```

Comment:
Nit - some extra 4s :)

Reply:
My pet SA is named Contessa and she walks on my keyboard sometimes :)